enumeratum-doobie/src/main/scala/enumeratum/Doobie.scala

```diff
@@ -1,6 +1,7 @@
 package enumeratum
 
 import doobie.util._
+import doobie.Meta
```

Comment:
Interesting ... a *patch* version bump resulted in breaking source compatibility??

Reply:
I think it broke binary compatibility as well.. :/ My code wouldn't compile because enumeratum-doobie is on 0.8.6 while I upgraded to 0.8.8. (Complains about missing doobie.util.Meta in classpath)

Reply:
Yikes. Thanks.